azure-tools-common/src/main/java/com/microsoft/azure/common/function/handlers/AnnotationHandlerImpl.java

```diff
@@ -84,10 +92,25 @@ public FunctionConfiguration generateConfiguration(final Method method) {
 
         patchStorageBinding(method, bindings);
 
+        config.setRetry(getRetryConfigurationFromMethod(method));
         config.setEntryPoint(method.getDeclaringClass().getCanonicalName() + "." + method.getName());
         return config;
     }
 
+    private Retry getRetryConfigurationFromMethod(Method method) throws AzureExecutionException {
+        final FixedDelayRetry fixedDelayRetry = method.getDeclaredAnnotation(FixedDelayRetry.class);
+        final ExponentialBackoffRetry exponentialBackoffRetry = method.getDeclaredAnnotation(ExponentialBackoffRetry.class);
+        if (fixedDelayRetry != null && exponentialBackoffRetry != null) {
+            throw new AzureExecutionException(MULTI_RETRY_ANNOTATION);
+        } else if (fixedDelayRetry != null) {
+            return Retry.createFixedDelayRetryFromAnnotation(fixedDelayRetry);
+        } else if (exponentialBackoffRetry != null) {
+            return Retry.createExponentialBackoffRetryFromAnnotation(exponentialBackoffRetry);
+        } else {
+            return null;
+        }
```

Comment:
Iprefer to:

if (fixedDelayRetry != null) {
            return Retry.createFixedDelayRetryFromAnnotation(fixedDelayRetry);
 }
if (exponentialBackoffRetry != null) {
            return Retry.createExponentialBackoffRetryFromAnnotation(exponentialBackoffRetry);
}
return null;

It's up to you.

Reply:
Fixed